frontend/src/pages/AccountSettingPage.tsx

```diff
@@ -1,31 +1,138 @@
+    event: React.MouseEvent<HTMLDivElement, MouseEvent>,
+    formType: "changename" | "changepassword",
+  ) => {
+    setFormType(formType);
+  };
 
   return (
-    <>
-      <UpdatePasswordModal
-        userId={user?.userId || 0}
-        open={isOpen}
-        onClose={onClose}
-      />
-      <Button variant="contained" onClick={onOpen}>
-        Update Password
-      </Button>
-    </>
+    <Paper elevation={1} sx={{ borderRadius: 3, mx: 6 }}>
+      <Stack direction="row">
+        <List>
+          <ListItem disablePadding sx={{ bgcolor: "white" }}>
+            <ListItemButton
+              selected={formType === "changename"}
+              onClick={(event) => handleListItemClick(event, "changename")}
+              sx={{
+                "&.Mui-selected": {
+                  backgroundColor: "white",
+                  color: "primary.500",
+                },
+              }}
+            >
+              <ListItemIcon>
+                <AccountCircle
+                  sx={{
+                    color:
+                      formType === "changename"
+                        ? "primary.500"
+                        : "blueGrey.500",
+                  }}
+                />
+              </ListItemIcon>
+              <ListItemText primary="Display Name" />
+            </ListItemButton>
+          </ListItem>
+          <ListItem disablePadding>
+            <ListItemButton
+              selected={formType === "changepassword"}
+              onClick={(event) => handleListItemClick(event, "changepassword")}
+              sx={{
+                "&.Mui-selected": {
+                  backgroundColor: "white",
+                  color: "primary.500",
+                },
+              }}
+            >
+              <ListItemIcon>
+                <Lock
+                  sx={{
+                    color:
+                      formType === "changepassword"
+                        ? "primary.500"
+                        : "blueGrey.500",
+                  }}
+                />
+              </ListItemIcon>
+              <ListItemText primary="Password" />
+            </ListItemButton>
+          </ListItem>
+        </List>
+        <Divider orientation="vertical" flexItem sx={{ mr: "-1px" }} />
+        {formType === "changename" ? (
+          <Stack spacing={4} sx={{ py: "20px", pl: 6, width: "60%" }}>
+            <Stack spacing={0.5}>
+              <Typography variant="body1" sx={{ fontWeight: "bold" }}>
+                Update Display Name
+              </Typography>
+              <Typography variant="body2">Personalize Your Name</Typography>
+              <Typography variant="body2">
+                Display Name must contain:
+              </Typography>
+            </Stack>
+            <Stack
+              direction="row"
+              sx={{ justifyContent: "center" }}
+              spacing={3}
+            >
+              <PasswordRequirement label="8+" description="Character" />
```

Comment:
We don't actually have any restrictions on the name, other than that it cannot be empty right now.
```suggestion
              <PasswordRequirement label="3+" description="Character" />
```